Assignment5.txt

```diff
@@ -0,0 +1,15 @@
+Priyanka Musti
+
+1.
+
+Passing by value= Pass the value of the variable of the variable into the fuction.
```

Comment:
sort of - you should have been more detailed. specifically, passing by value means passing a copy of the original value.